fix(curriculum): test in password generator project, step 54 (#54804)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/07-scientific-computing-with-python/learn-regular-expressions-by-building-a-password-generator/6564f834dd717998092cfd47.md

> You should write a `for` loop to iterate over the `constraints` list with `constraint` and `pattern`.

Reviewer: Both hints seem quite similar. The first one can be updated as:

```suggestion
You should write a `for` loop that uses the `constraints` list as the loop variable.
```

Author: I forgot to change the first after adding the second one.
